app/controllers/content_items_controller.rb

```diff
@@ -7,11 +7,18 @@ class ContentItemsController < ApplicationController
   def show
     load_content_item
     set_expiry
+    @ab_testing_group = ab_testing_group
     with_locale do
       render content_item_template
     end
   end
 
+  def should_present_b_view?
+    (ab_testing_group == "B") && new_navigation_enabled? && content_is_tagged_to_a_taxon?
```

Comment:
Do we have the new chrome extension ready so we can add ourselves to the B group in the A/B test on integration?